src/app/games/encapsule/encapsule.component.ts

```diff
@@ -171,20 +180,17 @@ export class EncapsuleComponent extends RectangularGameComponent<EncapsuleRules,
     }
 
     public getPieceRadius(piece: EncapsulePiece): number {
-        switch (piece.getSize()) {
-            case Size.BIG:
-                return (this.SPACE_SIZE / 2) - (1 * this.STROKE_WIDTH);
-            case Size.MEDIUM:
-                return (this.SPACE_SIZE / 2) - (2 * this.STROKE_WIDTH) - 3;
-            default:
-                Utils.expectToBe(piece.getSize(), Size.SMALL);
-                return (this.SPACE_SIZE / 2) - (3 * this.STROKE_WIDTH) - 6;
-        }
+        const size: number = piece.getSize();
+        // The aim is that the inter-piece space is half a piece stroke
+        // And the radius is used like this (from center to outside):
+        // empty - stroke - stroke - empty - stroke - stroke
+        // And stick to the square's inner-edge
```

Comment:
une phrase ne peut pas commencer par "And", et ceci n'est pas une phrase complète

Reply:
trouve la formulation j'ai zéro idée